flight/Modules/System/systemmod.c

```diff
@@ -953,6 +953,7 @@ static int32_t eventPeriodicCreate(UAVObjEvent* ev, UAVObjEventCallback cb, stru
 	objEntry->evInfo.ev.obj = ev->obj;
```

Comment:
This should help a lot with other constrained links like BTLE, which I revived yesterday.